app/services/articles/ranked_and_sorted.rb

```diff
@@ -0,0 +1,37 @@
+module Articles
+  class RankedAndSorted
```

Comment:
I wonder if these new files shouldn't be in `Articles::Feed::RankedAndSorted`

cc. @joshpuetz

Reply:
Yes, agreed @rhymes: that would fit our general naming patterns a bit better, and convey that this module is related to the _feed_ specifically